xbmc/pvr/windows/GUIViewStatePVR.cpp

```diff
@@ -77,8 +77,11 @@ void CGUIViewStateWindowPVRGuide::SaveViewState(void)
 
 CGUIViewStateWindowPVRTimers::CGUIViewStateWindowPVRTimers(const int windowId, const CFileItemList& items) : CGUIViewStatePVR(windowId, items)
 {
-  AddSortMethod(SortByLabel, 551, LABEL_MASKS("%L", "%I", "%L", ""));   // FileName, Size | Foldername, empty
-  AddSortMethod(SortByDate, 552, LABEL_MASKS("%L", "%J", "%L", "%J"));  // FileName, Date | Foldername, Date
+  SortAttribute sortAttributes(SortAttributeIgnoreFolders);
+  if (CSettings::Get().GetBool("filelists.ignorethewhensorting"))
+    sortAttributes = (SortAttribute) (sortAttributes | SortAttributeIgnoreArticle);
```

Comment:
I would love to but it won't compile
```code
GUIViewStatePVR.cpp: In constructor 
‘PVR::CGUIViewStateWindowPVRTimers::CGUIViewStateWindowPVRTimers(int, const CFileItemList&)’:
GUIViewStatePVR.cpp:83:20: error: invalid conversion from ‘int’ to ‘SortAttribute’ [-fpermissive]
     sortAttributes |= SortAttributeIgnoreFolders;
                    ^
```
I'm afraid this is beyond my CPP skills. Could you help?